govcd/system_test.go

```diff
@@ -833,3 +833,83 @@ func (vcd *TestVCD) Test_NsxtGlobalDefaultSegmentProfileTemplate(check *C) {
 	err = createdSegmentProfileTemplate.Delete()
 	check.Assert(err, IsNil)
 }
+
+// Test retrieval of all Orgs
+func (vcd *TestVCD) Test_QueryAllOrgs(check *C) {
+	vcd.skipIfNotSysAdmin(check)
+	if vcd.config.VCD.Org == "" {
+		check.Skip("Test_QueryOrgByName: Org Name not given")
+		return
+	}
+
+	orgs, err := vcd.client.QueryAllOrgs()
+	check.Assert(err, IsNil)
+	check.Assert(orgs, NotNil)
+
+	if vcd.config.VCD.Org != "" {
```

Comment:
This looks to be unnecessary as the test is skipped if Org is not present a few lines above

Reply:
Correct. Removed